invenio_records_lom/fixtures/demo.py

```diff
@@ -0,0 +1,363 @@
+    }
+
+
+def create_fake_datetime(fake: Faker) -> dict:
+    """Create a fake DateTime-object, compatible with LOMv1.0-standard."""
+    pattern = fake.random.choice(["YMDhmsTZD", "YMDhms", "YMD", "Y"])
+    if pattern == "Y":
+        datetime = fake.year()
+    elif pattern == "YMD":
+        datetime = fake.date()
+    elif pattern == "YMDhms":
+        datetime = fake.date_time().isoformat()
+    elif pattern == "YMDhmsTZD":
+        tzd = fake.random.choice(["UTC", "CST", "JST"])
+        datetime = fake.date_time().isoformat() + tzd
+
+    return {"dateTime": datetime, "description": langstringify(fake, fake.sentence())}
+
+
+def create_fake_duration(fake: Faker) -> dict:
+    """Create a fake Duration-object, compatible with LOMv1.0-standard."""
+    random = fake.random
+    pattern = random.choice(["all", "Y", "D", "HM", "S"])
+    if pattern == "all":
+        duration = "P1Y2M4DT10H35M12.5S"
+    elif pattern == "Y":
+        duration = f"P{random.randint(1,5)}Y"
+    elif pattern == "D":
+        duration = f"P{random.randint(1,60)}D"
+    elif pattern == "HM":
+        duration = f"PT{random.randint(1,3)}H{random.randint(1,59)}M"
+    else:
+        duration = f"PT{random.uniform(0.1, 12.5)}S"
+
+    return {
+        "duration": duration,
+        "description": langstringify(fake, fake.sentence()),
+    }
+
+
+def create_fake_vcard(fake: Faker) -> str:
+    """Returns a placeholder-string for a vCard-object."""
+    return "placeholder for vcard"
+
+
+# ----- functions for elements that are part of more than one category -----
+def create_fake_language(fake: Faker) -> str:
+    """ "Create a fake language-code, as required for "language"-keys by LOMv1.0-standard."""
+    return fake.language_code()
+
+
+def create_fake_identifier(fake: Faker) -> dict:
+    """Create a fake "identifier"-element, compatible with LOMv1.0-standard."""
+    catalog = fake.random.choice(["URI", "ISBN"])
+    if catalog == "URI":
+        entry = fake.uri()
+    else:
+        entry = fake.isbn13()
+
+    return {
+        "catalog": catalog,
+        "entry": entry,
+    }
+
+
+def create_fake_contribute(fake: Faker, roles: list) -> dict:
+    """Create a fake "contribute"-element, compatible with LOMv1.0-standard."""
+    return {
+        "role": vocabularify(fake, roles),
+        "entity": [create_fake_vcard(fake) for __ in range(2)],
+        "date": create_fake_datetime(fake),
+    }
+
+
+# ----- functions for categories or used by only one category -----
+def create_fake_general(fake: Faker) -> dict:
+    """Create a fake "general"-element, compatible with LOMv1.0-standard."""
+    structures = ["atomic", "collection", "networked", "hierarchical", "linear"]
+    aggregationLevels = [1, 2, 3, 4]
+
+    return {
+        "identifier": [create_fake_identifier(fake) for __ in range(2)],
+        "title": langstringify(fake, " ".join(fake.words())),
+        "language": [fake.random.choice([create_fake_language(fake), "none"])],
```

Comment:
is language really a array?

Reply:
Yes, it's explicitly stated in the LOMv1.0 specification:
![image](https://user-images.githubusercontent.com/86989418/128828466-cff001ee-baab-44cb-99e1-780f356a3a00.png)